frontend/src/components/taskmanager/addTask.tsx

```diff
@@ -0,0 +1,47 @@
+import React, { useState } from 'react';
+import './addTask.css';
+
+const AddTask: React.FC = () => {
+  const [task, setTask] = useState("");
+ 
+  const handleSubmit = async (e: React.FormEvent<HTMLFormElement>) => {
```

Comment:
write proper services, hooks for these data related handling, component should only be concerned about rendering the page